Sources/Fluent/Query/Filter.swift

```diff
@@ -129,3 +132,15 @@ extension QueryRepresentable {
         return try filter(T.self, field, .contains, value)
     }
 }
+
+extension QueryRepresentable {
+    @discardableResult
+    public func raw(command: String, values: [NodeRepresentable] = []) throws -> Query<Self.T> {
```

Comment:
I don't fully understand this comment, so @Casperhr please weigh in if this is the type of level you're looking for.